web/templates/presence_row.hbs

```diff
@@ -4,11 +4,17 @@
             <span class="{{user_circle_class}} user_circle"></span>
             <a class="user-presence-link" href="{{href}}">
                 <div class="user-name-and-status-wrapper">
+                    {{#if status_text}}
+                    <div class="user-name-and-status-emoji">
```

Comment:
`user-name-and-status-emoji` is a misleading classname now, maybe rename the classname globally to just `user-name`? Or maybe just change the classname here and have it share styles with `user-name-and-status-emoji`?